BR Simple/DetailsViewController.swift

```diff
@@ -6,12 +6,56 @@
 //
 
 import UIKit
+import MapKit
 
-final class DetailsViewController: UIViewController {
+final class DetailsViewController: UIViewController, MKMapViewDelegate {
+    @IBOutlet var image: UIImageView!
+    @IBOutlet var name: UILabel!
+    @IBOutlet var category: UILabel!
+    @IBOutlet var address: UILabel!
+    @IBOutlet var phone: UILabel!
+    @IBOutlet var social: UILabel!
+    @IBOutlet var mapKitView: MKMapView!
```

Comment:
naming convention should be followed for example var nameLabel: UILabel, var imageView: UIImageView